ERCS/erc-7699.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 7699
+title: Soul Resonance Token
+description: this EIP defines an interface for Soul Resonance Tokens
```

Comment:
Your description just repeats your title. Try to expand on the idea you introduce there, instead of just restating it. Perhaps explain what a Soul Resonance Token is?